packages/json/json.pony

```diff
@@ -75,7 +85,17 @@ class JsonObject
         text = text + ","
       end
 
-      text = text + "\n" + elem_indent + "\"" + i._1 + "\": " +
-        _JsonPrint._string(i._2, elem_indent)
+      if pretty_print then
+        text = text + "\n" + elem_indent + "\"" + i._1 + "\": " +
+          _JsonPrint._string(i._2, elem_indent, true)
+      else
+        text = text + if text.size() > 0 then " " else "" end
+          + "\"" + i._1 + "\": " +
```

Comment:
The space-after-colon should be removed in not-pretty printing to save the byte.